pypeclub/OpenPype
# Dirmap in Nuke

Implements same dirmap functionality as in Maya for Nuke.
(Takes into account Site Sync and local sites.)

Needs to be tested cross platform
![Screenshot 2021-11-02 153606](https://user-images.githubusercontent.com/4457962/139868446-35d4c35f-04a8-44f9-a103-0934593f6e00.png)
!

How to test it (how I tested it):
----------------------------------
- prepare Nuke workfile with referenced file (use `Load` to import some asset)
- create `new destination` folder on your machine
- move physical referenced file that are used in workfile from `old` folder to `new destination`
- without dirmap when you open Nuke Read node of referenced files should complain that is not seeing referenced file
- enable dirmap and set mapping in Project Settings (from existing to new destination folder)
- run Tray and Nuke again
- this time Nuke should automatically pick file from `new destination`
- links doesn't change in Read nodes! (thats by default from Nuke)
